fiftyone/core/odm/database.py

```diff
@@ -501,6 +501,16 @@ def get_db_conn():
     return _apply_options(db)
 
 
+def get_db_version():
+    """Returns the database version.
```

Comment:
This is checking the same version as https://github.com/voxel51/fiftyone/blob/5872c4f3ec80a0f17b4aa78b16ec025d0035e8ba/fiftyone/core/odm/database.py#L343. Should that function be updated?

Reply:
Like to reduce code duplication? For me the shared code:

```py
Version(client.server_info()["version"])
```

doesn't quite reach the bar of needing a util method, as its just one line. If it were multiple lines of logic, I could see it.

Reply:
Yeah like just update that code to call this one in case it changes for whoever reason.